test/mongoGetCommand.test.ts

```diff
@@ -934,6 +932,23 @@ suite("scrapbook parsing Tests", () => {
         });
     });
 
+    test("Chained command where final function call is recognized. Ensure execution is sent to shell - test user issues: https://github.com/Microsoft/vscode-cosmosdb/issues/981", () => {
+        testParse(`db.getCollection('my-collection').find({});`, {
```

Comment:
What about this checks that it gets sent to the shell?

Reply:
The command is chained - MongoFindResultEditor cannot handle this.

Reply:
> What about this checks that it gets sent to the shell?

The collection name is undefined. A command like `db.abc.find()` would have the collection name be abc.

Reply:
I agree with Stephen that the name of this test isn't accurate. I would just name it something like `find command for collection with hyphens in name, from #981`

If you want to create a test that represents the name you currently have, it should somehow actually verify against the shell. But I don't think that's worth doing until after https://github.com/microsoft/vscode-cosmosdb/pull/1105